src/classes/AccountHandler.cls

```diff
@@ -0,0 +1,20 @@
+public class AccountHandler {
+    public static Account insertNewAccount(string accName) {
+        Account acct = new Account(Name = accName);
+        
+        try {
+            insert acct;
+        }
+        catch (DmlException error) {
+            System.debug('DML exception occured' + error.getMessage());
+            return null;
+        }
+        return acct;
+    }
+}
+
+/*
+Execution - AccountHandler.insertNewAccount('Zaheer1');
+
+Output - 13:38:02:014 USER_DEBUG [6]|DEBUG|Executed
```

Comment:
Where is the debug statement, you have added it though 6 line I cant see it.
you have to add debug in a way like 
Ex: System.debug(acct.Id); at line after 12
so it will print id after insert